lib/vetext/README.md

```diff
@@ -0,0 +1,61 @@
+# VEText Push Notifications API
+version 1.1 beta
+
+## Definitions
+
+### SID
+An AWS term; "Statement ID"
+
+### App SID
+Each mobile application has a master id recored in the VEText system. This is called the App SID, and it's configures inside of vets-api.
+
+### Endpoint
+A single mobile app installation. Keyed on device token, ICN.
+
+### Endpoint SID
+Each mobile application installation has a master id recorded in the VEText system. This is called the Endpoint SID. The mobile client is expected to store this for future use in fetching and updating Preferences.
+
+An Endpoint has many Preferences
+
+### Token (device token)
+The unique token representing a device, generated by Apple or Google
+
+### Channel
+Email, SMS, push – each is a channel of communication. Not used formally in this api, but valuable to define it in this context.
+
+### Preference
+A definition of a notification type. When paired with an Endpoint, a preference describes the user's desire to receieve that notification type.
+
+A Preference has many Templates.
+
+A Preferense belongs to many Endpoints.
+
+### Preference_id (Notification Type)
+A string that describes the notification type
+
+### Template
+A string template for the push message, associated with a preference. A Preference may have multiple Templates, but only one is set to the the default.
+
+A Template belongs to a Preference.
+
+## Endpoints
+
+### Register a mobile app
+PUT /api/vetext/pub/mobile/push/endpoint
+
+Send a device token, ICN, and App SID, get back an Endpoint SID.
+
+### Get a user's prefs
+GET /api/vetext/pub/mobile/push/preference/client/{endpoint_sid}
+
+Get a list of Preferences for a given Endpoint, along with whether the user wants to receive them.
+
+### Set a user's pref
+PUT /api/vetext/pub/mobile/push/preference/client
+
+Set a Preference to true or false, whether the user wants to receive them. Call with one Preference at a time.
+
+### Send a push notification
+POST /api/vetext/pub/mobile/push/send
+
+Send a push notification. Pass Endpoint SID or ICN as well a Preference SID.
```

Comment:
Consider merging this info into the api docs

Reply:
All of this info is more or less part of the mobile API docs, or it will be when I add the send endpoint. I don't see too much reason for this file anymore. I'm happy to get rid of it if that's what we'd prefer.

Reply:
I like the top half of the file, which will allow us to "translate" some of the abstract concepts from the VEText API

Reply:
sounds good, I'll leave the dictionary portion and remove the endpoints since I think that should be covered by API documentation anyway.